### DISCUSSION TITLE
Transferring npins to nix-community

### DISCUSSION THREAD

**Discussion by fricklerhandwerk** [nix-community/infra#1300] [Q&A] [answered]
Flakes are experimental, niv is not maintained any more, but @andir's npins works, has very few moving parts, is tended to by at least two people, and is known to be in use by many prolific contributors.

I think it would be appropriate to give the project more visibility.

@andir seems to have been in favor of graduating it to nix-community last time we talked. 

Related: [defining and curating official projects](https://discourse.nixos.org/t/call-for-volunteers-curating-official-projects/45382)

What needs to be done to make the move? Any blockers? And who's currently in charge of such administrative decisions in nix-community?

**Comment by zowoq** [comment, marked as answer]
> What needs to be done to make the move? Any blockers? 

None, `@andir` is already a member of @nix-community and members have permissions to transfer repos into the org.

> And who's currently in charge of such administrative decisions in nix-community?

https://nix-community.org/administrators/

After it is transferred we'll restore the original owners permissions on the project as github usually downgrades them and we'll create a team for the project as well.

For transferring projects we only really have one "rule":

> https://github.com/nix-community/infra/issues/534#issuecomment-1514786986
> Ideally, the org holds projects that are more than MVPs, although we didn't really discuss what standards we want to hold here. The issue with too many MVP repos is that it creates a graveyard of unmaintained projects.

A couple of times we've disabled permissions for creating/transferring repos because it was used for new projects but it ended up being a hassle to keep switching it on/off and keeping track of it.
